bin/strata-reth/src/rpc.rs

```diff
@@ -31,18 +31,21 @@ impl From<SequencerRpcError> for EthApiError {
 
 /// A client to interact with a Sequencer
 #[derive(Debug, Clone)]
+#[allow(dead_code)] // FIXME: remove this
```

Comment:
yes, my bad :( I removed it already in the upcoming PR (bumping reth to latest)

Reply:
Ok, once we have it up we can link the PRs by adding the number here?

Reply:
I am inclined to put this PR before #542 since it depends on succinct labs bumping their cargo version.

What you think? You can rebase and remove the `dead_code` lints.